daemons/compute/server/servers/server_sim.go

```diff
@@ -50,29 +51,56 @@ func (s *simulatedServer) Create(ctx context.Context, req *pb.DataMovementCreate
 	return &pb.DataMovementCreateResponse{Uid: uid.String()}, nil
 }
 
+var statusCount int = 1
```

Comment:
Should we make this part of the `s.requests[]` map instead of a global variable?